### PR TITLE
Remove advertisement of Get-Error experimental feature from module manifest

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

The `Get-Error` experimental feature code was removed in the .cs files, but missed in the .psd1 file.  `Get-Error` still works, but is incorrectly advertised via `Get-ExperimentalFeature` as being experimental.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1 [hunk 1]
```diff
@@ -38,10 +38,6 @@ PrivateData = @{
         Name        = 'Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace'
```
Comment (PoshChan) [inline src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: @SteveL-MSFT, your last commit had 47 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 272, but got 271.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 21
21:         $types.Count | Should -BeExactly ($IsWindows ? 273 : 272)
```
TabCompletion.Should complete dotnet method with null conditional operator
```powershell
Expected exactly 'ToString(', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 38
38:         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'ToString('
```

Comment (PoshChan) [inline src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: @SteveL-MSFT, your last commit had 44 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
TabCompletion.Should complete dotnet method with null conditional operator
```powershell
Expected exactly 'ToString(', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 38
38:         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'ToString('
```
TabCompletion.Should complete dotnet method with null conditional operator without first letter
```powershell
Expected exactly 'CompareTo(', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 43
43:         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'CompareTo('
```

Comment (PoshChan) [inline src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: @SteveL-MSFT, your last commit had 47 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 272, but got 271.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 21
21:         $types.Count | Should -BeExactly ($IsWindows ? 273 : 272)
```
TabCompletion.Should complete dotnet method with null conditional operator
```powershell
Expected exactly 'ToString(', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 38
38:         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'ToString('
```

**Comment by daxian-dbw** [comment]
close/open the PR to re-trigger CI runs.

File: src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1 (hunk 1 above)
Comment (PoshChan) [inline src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: @SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Exception calling "Invoke" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 136
136:                 $ps.AddScript('$pid').Invoke() | Should -Be $pwshId

```

**Comment by SteveL-MSFT** [comment]
@PoshChan please retry windows

**Comment by PoshChan** [comment]
@SteveL-MSFT, successfully started retry of `PowerShell-CI-Windows`

File: src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1 [hunk 2]
```diff
@@ -37,10 +37,6 @@ PrivateData = @{
         Name        = 'Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace'
         Description = 'Enables -BreakAll parameter on Debug-Runspace and Debug-Job cmdlets to allow users to decide if they want PowerShell to break immediately in the current location when they attach a debugger.'
       }
-      @{
-        Name        = 'Microsoft.PowerShell.Utility.PSGetError'
-        Description = 'Enable Get-Error cmdlet that displays detailed information about ErrorRecords included nested objects'
```
Comment (anmenaga) [inline src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: Why this section in psd1 was necessary?

Comment (SteveL-MSFT) [reply src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: I believe this probably should have been only in the psd1 and not in the internal list of experimental features

Comment (daxian-dbw) [reply src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: Yes, the internal list of experimental features are supposed to be engine features, and the `Source` would be `PSEngine`. For experimental features exposed from a module, it should be declared in the psd1 file, not in the engine feature list.

**Review by anmenaga — APPROVED** [review]

**Pull request merged**